Library/Homebrew/test/dev-cmd/extract_spec.rb

```diff
@@ -14,36 +14,49 @@
       core_tap = CoreTap.new
       core_tap.path.cd do
         system "git", "init"
-        formula_file = setup_test_formula "testball"
+        # Start with deprecated bottle syntax
+        setup_test_formula "testball", bottle_block: <<~EOS
+
+          bottle do
+            cellar :any
+          end
+        EOS
         system "git", "add", "--all"
         system "git", "commit", "-m", "testball 0.1"
+        # Remove it for next version
```

Comment:
Just a non-blocking suggestion to clarify the comment

```suggestion
        # Replace the contents with a valid formula for the next version
```